inc/networkportinstantiation.class.php

```diff
@@ -277,17 +277,21 @@ function getInstantiationHTMLTable(NetworkPort $netport, HTMLTableRow $row,
 
          $virtual_header = $row->getHeaderByName('Instantiation', 'VirtualPorts');
 
-         $query = "(SELECT `networkports_id`
-                    FROM `glpi_networkportaliases`
-                    WHERE `networkports_id_alias`='".$netport->getID()."')
-                   UNION
-                   (SELECT `networkports_id`
-                    FROM `glpi_networkportaggregates`
-                    WHERE `networkports_id_list` LIKE '%\"".$netport->getID()."\"%')";
-
-         $iterator = $DB->request($query);
-
-         if ($iterator->numrows() > 0) {
+         $iterator = $DB->request([
+            'FROM' => new \QueryUnion(
+               [
+                  'SELECT' => 'networkports_id',
+                  'FROM'   => 'glpi_networkportaliases',
+                  'WHERE'  => ['networkports_id_alias' => $netport->getID()]
+               ], [
+                  'SELECT' => 'networkports_id',
+                  'FROM'   => 'glpi_networkportaggregates',
+                  'WHERE'  => ['networkports_id_list' => ['LIKE', '%'.$netport-getID().'%']]
```

Comment:
Value was surrounded by `"`. Is this intentionnal to remove these quotes ?

Reply:
nope, good catch